presentation/valid/ttml2-prstn-font-shear-linear-mapping.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tt xml:lang="ja"
+    xmlns="http://www.w3.org/ns/ttml"
+    xmlns:ttm="http://www.w3.org/ns/ttml#metadata"
+    xmlns:ttp="http://www.w3.org/ns/ttml#parameter"
+    xmlns:tts="http://www.w3.org/ns/ttml#styling"
+    ttp:profile="ttml2-presentation"
+    xmlns:ttva="http://skynav.com/ns/ttv/annotations" ttva:warnOn="all" ttva:expectedErrors="0" ttva:expectedWarnings="0" ttva:model="ttml2">
+  <head>
+    <metadata>
+      <ttm:desc>#fontShear - linear mapping</ttm:desc>
+    </metadata>
+    <styling>
+      <style xml:id="sRegion" tts:showBackground="whenActive" tts:backgroundColor="black" tts:displayAlign="center"/>
+      <style xml:id="sText" tts:color="white" tts:textAlign="center" tts:fontSize="6.67rh" tts:lineHeight="8.0rh"/>
+    </styling>
+    <layout>
+      <region xml:id="horizontal" style="sRegion sText" tts:extent="40% 60%" tts:origin="5% 5%"/>
+      <region xml:id="vertical" style="sRegion sText" tts:origin="55% 5%" tts:extent="40% 60%" tts:writingMode="tblr"/>
+      <region xml:id="caption" style="sRegion sText" tts:origin="10% 75%" tts:extent="80% 20%"/>
+    </layout>
+  </head>
+  <body>
+    <div begin="0s" end="1s">
+      <p region="caption">Zero font shear</p>
+      <p region="horizontal" tts:fontShear="0%">0%<br/>三日坊主</p>
+      <p region="vertical" tts:fontShear="0%">0%<br/>三日坊主</p>
+    </div>
+    <div begin="1s" end="2s">
+      <p region="caption">Positive font shear</p>
+      <p region="horizontal" tts:fontShear="16.78842%">16.78842%<br/>三日坊主</p>
```

Comment:
Out of curiosity, why was this value chosen? It maps to 15.109758deg. We could use 16.667% we would map to ~15deg.

Reply:
Because of historical usage with Lambda CAP. I think we've already had this discussion before. I'd have to dig into my archives to pull out the details.